License2Deploy/AWSConn.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+
+import boto.ec2 as ec2
+import boto.ec2.autoscale as a
+import boto.ec2.elb as elb
+
+class AWSConn(object):
+
+  @staticmethod
+  def aws_conn_auto(region, profile='default'):
+    try:
+      conn = a.connect_to_region(region, profile_name=profile)
+      return conn
+    except Exception as e:
+      print("Unable to connect to region, please investigate: {0}".format(e))
+
+  @staticmethod
+  def aws_conn_ec2(region, profile='default'):
+    try: 
+      conn = ec2.connect_to_region(region, profile_name=profile)
+      return conn
+    except Exception as e:
+      print("Unable to connect to region, please investigate: {0}".format(e))
+
+  @staticmethod
+  def aws_conn_elb(region, profile='default'):
+    try:
+      conn = elb.connect_to_region(region, profile_name=profile)
+      return conn
+    except Exception as e:
+      print("Unable to connect to region, please investigate: {0}".format(e))
+
+  @staticmethod
+  def determine_region(env):
+    if env == 'qa':
+      return 'us-west-1'
+    elif env in ['stg', 'prd']:
+      return 'us-east-1'
+    else:
+      return env # This method is somewhat specific to our environment, so I am allowing the option for env to equal an actual region and will return the original value supplied
```

Comment:
@adamjkeller   could we use config file for this method, so it will be useful for other people.

Also, method should return False or valid(!) AWS region name, like ``us-east-1``